### DISCUSSION TITLE
Designation of Tracking Domains for Privacy Manifest Compliance

### DISCUSSION THREAD

**Discussion by tyomashin** [firebase/firebase-ios-sdk#12224] [Q&A]
I am currently working on a privacy manifest for my application.

According to the following documentation, since Firebase Analytics may access IDFA,
user permissions for tracking by the ATT framework in the application.

* https://firebase.google.com/docs/ios/supporting-ios-14#affected-firebase-products

In the future, apps that require tracking in the ATT framework,
I am aware that I need to specify the tracking domain in the privacy manifest.
I am going to specify the Analytics domain as this domain.

Could you please provide us with the Analytics domain?
If there are any discrepancies in the perceptions expressed here, please point them out to us.

**Comment by mmkhmk** [comment]
The discussion seems ongoing in this ticket.
https://github.com/firebase/firebase-ios-sdk/issues/11490

**Comment by paulb777** [comment]
While we're still working on updates for the new required reason APIs, nutrition label information can be found at https://firebase.google.com/docs/ios/app-store-data-collection
